migrations/hasura/migrations/default/1680330156157_Init/up.sql

```diff
@@ -8164,3 +8164,24 @@ ALTER TABLE ONLY public.voucher_plan_product
     ADD CONSTRAINT voucher_plan_product_voucher_plan_id_fkey FOREIGN KEY (voucher_plan_id) REFERENCES public.voucher_plan(id) ON UPDATE RESTRICT ON DELETE RESTRICT;
 ALTER TABLE ONLY public.voucher
     ADD CONSTRAINT voucher_voucher_code_id_fkey FOREIGN KEY (voucher_code_id) REFERENCES public.voucher_code(id) ON UPDATE RESTRICT ON DELETE RESTRICT;
+CREATE TABLE "public"."program_content_ebook" ("id" uuid NOT NULL DEFAULT gen_random_uuid(), "data" jsonb NOT NULL, "attachment_id" uuid NOT NULL, "created_at" timestamptz NOT NULL DEFAULT now(), "updated_at" timestamptz NOT NULL DEFAULT now(), PRIMARY KEY ("id") );
+CREATE OR REPLACE FUNCTION "public"."set_current_timestamp_updated_at"()
+RETURNS TRIGGER AS $$
+DECLARE
+  _new record;
+BEGIN
+  _new := NEW;
+  _new."updated_at" = NOW();
+  RETURN _new;
+END;
+$$ LANGUAGE plpgsql;
+CREATE TRIGGER "set_public_program_content_ebook_updated_at"
+BEFORE UPDATE ON "public"."program_content_ebook"
+FOR EACH ROW
+EXECUTE PROCEDURE "public"."set_current_timestamp_updated_at"();
+COMMENT ON TRIGGER "set_public_program_content_ebook_updated_at" ON "public"."program_content_ebook"
+IS 'trigger to set value of column "updated_at" to current timestamp on row update';
+alter table "public"."program_content_ebook" rename column "attachment_id" to "program_content_id";
```

Comment:
The 'ebook' and 'attachment' tables are not related

Reply:
Hi @Ponchimeow 

`8167` - `8183` is create table program_content_ebook and include a column name `attachment_id`
 > source migration:  1702628692539_create_table_public_program_content_ebook 

`8184` is alter table column `attachment_id` to `program_content_id`
> source migration:  1702629092033_alter_table_public_program_content_ebook_alter_column_attachment_id

i think these two migration will not casuse `ebook` and `attachment` related
The reason why I didn't change the migration of create table directly is because I wanted to maintain consistency with the source migration(`lodestar-app-hasura`).